.github/workflows/test-integrations.yml

```diff
@@ -538,6 +544,10 @@ jobs:
         run: chmod +x consul
       - name: Build consul:local image
         run: docker build -t ${{ env.CONSUL_LATEST_IMAGE_NAME }}:local -f ./build-support/docker/Consul-Dev.dockerfile .
+      - name: Build consul-envoy:latest-version image
+        run: docker build -t consul-envoy:latest-version --build-arg CONSUL_IMAGE=docker.mirror.hashicorp.services/${{ env.CONSUL_LATEST_IMAGE_NAME }}:${{ env.CONSUL_LATEST_VERSION }} --build-arg ENVOY_VERSION=${{ env.ENVOY_VERSION }} -f ./test/integration/consul-container/assets/Dockerfile-consul-envoy ./test/integration/consul-container/assets
+      - name: Build consul-envoy:target-version image
+        run: docker build -t consul-envoy:target-version --build-arg CONSUL_IMAGE=${{ env.CONSUL_LATEST_IMAGE_NAME }}:local --build-arg ENVOY_VERSION=${{ env.ENVOY_VERSION }} -f ./test/integration/consul-container/assets/Dockerfile-consul-envoy ./test/integration/consul-container/assets
```

Comment:
An editorial comment that doesn't require any action: I find the `latest` and `target` terminology unclear. I realize they are "before" and "after", respectively, but `latest` can also point to a legacy version. I don't have a strong opinion on replacement, maybe "base/target", "before/after", or "old/new". Just something that would be less ambiguous.

Reply:
Yeah, there are other places in the code where latest and target are used. (I think the name was introduced since the compatibility test was added to the repo : )